content/en/docs/rules/conditions.md

```diff
@@ -0,0 +1,59 @@
+Every syscall event will present the [`evt` field class](/docs/rules/supported-fields/#field-class-evt). Each condition that we write for those events will normally starts with a `evt.type` expression or macro; this makes a lot of sense since security rules normally consider one syscall type at a time. For instance, we may want to consider `open` or `openat` to catch suspicious activity when opening files, `execve` to inspect spawned processes and so forth. We don't have to guess the syscall name, as we can look at the complete [list of supported system calls events](/docs/rules/supported-events) or to understand which ones we can use.
+
+Every syscall has an entry event and an exit event which is shown in the `evt.dir` field, also known as the "direction" of the system call. A value of `>` indicates entry, which fires when the syscall is invoked, while `<` marks exit, meaning that the call has returned. In fact, by looking at the supported system call list we can see that our events have both entries. For example:
+
+```
+> setuid(UID uid)
+< setuid(ERRNO res)
+```
+
+Most of the times the engine will tell us a lot more information about exit events as we want to understand what happened after the event execution is complete as we can see with events concerning opening files.
+
+```
+> open()
+< open(FD fd, FSPATH name, FLAGS32 flags, UINT32 mode, UINT32 dev)
+> openat()
+< openat(FD fd, FD dirfd, FSRELPATH name, FLAGS32 flags, UINT32 mode, UINT32 dev)
+```
+
+As we can see, each event has a list of arguments associated with it that we can access by using `evt.arg.<argname>`. For example, if we want to identify when a process opens a file to overwrite it we need to check if the list of flags contains `O_TRUNC` we will use the `evt.arg.flags` field of the `open` and `openat` exit events shown above and our rule will then look like:
+
+```
+evt.type in (open, openat) and evt.dir = < and evt.arg.flags contains O_TRUNC
+```
+
+Note that the arguments do not necessarily match the raw parameters that are used in the Linux kernel, but are parsed by Falco to make it easier to write rules. By using the [`evt` fields](/docs/rules/supported-fields/#field-class-evt) we can inspect many more aspects that are common across events.
+
+# Syscall event context and metadata
+
+While the `evt` fields allow us to write pretty expressive conditions, arguments and common fields are usually not enough to write full security rules. Many times we want to add conditions based on the process context the event happens in, or whether or not something is happening inside a container or even correlate each event with the relevant Kubernetes metadata for our cluster, pods, and more. For this reason Falco enriches many events with [other field classes](/docs/rules/supported-fields). Not all classes are available for all events and the list can grow. The documentation for each clarifies when those are expected to be available, but some are so common that we often rely on them.
```

Comment:
While the `evt` fields allow us to write pretty expressive conditions, arguments and common fields are usually not enough to write full security rules. Many times you want to add conditions based on the process context the event happens in, or whether or not something is happening inside a container or even correlate each event with the relevant Kubernetes metadata for the cluster, pods, and more. For this reason, Falco enriches many events with [other field classes](/docs/rules/supported-fields). Not all the classes are available for all the events and the list can grow. The documentation for each clarifies when those are expected to be available, but some are so common that you often rely on them.